README.Rmd

```diff
@@ -717,7 +717,7 @@ As in the previous attempt, the following code runs the validation using
 a GWR model.
 
 ```{r,eval=FALSE,echo=FALSE}
-val_results_gwr = lapply(networks_files,
+val_results_gwr = lapply(networks_files[1],
```

Comment:
Note @juanfonsecaLS1 I had to include only the first object because the second failed.